compose/ui/ui-test-junit4/src/skikoMain/kotlin/androidx/compose/ui/test/ComposeUiTest.skikoMain.kt

```diff
@@ -128,31 +128,6 @@ class SkikoComposeUiTest(
         override fun updateState(oldValue: TextFieldValue?, newValue: TextFieldValue) = Unit
     }
 
-    @OptIn(ExperimentalTextApi::class)
-    private val textInputServiceForTests = object : TextInputForTests {
```

Comment:
Is this test no more needed?

Reply:
This is not the test it was used by test, but now it works using another mechanism as I got from the Google folk's commit message

Reply:
Thanks! LGTM!